gigareview/console/tests/acceptance/FetchReports.feature

```diff
@@ -5,6 +5,11 @@ Feature: FetchReports
 
   Scenario: try FetchReports
     Given a EM report is uploaded daily to a sftp server
+    And the database is reset
```

Comment:
If you need the database to be in a clean state before tests then I would have thought the best place to implement this is in `acceptance.suite.yml` using a [Db](https://codeception.com/docs/modules/Db) module which Codeception says it should be used for.

Reply:
I have tried to include this to the acceptance.suite.yml file
```
      - Db:
                  dsn: 'pgsql:host=reviewdb;port=5432;dbname=reviewdb_test'
                  user: 'reviewdb'
                  password: 'reviewdb'
                  dump: 'sql/reviewdb_test.pgdmp'
                  cleanup: true
                  populate: true
                  populator: 'PGPASSWORD=reviewdb pg_restore -h reviewdb -p 5432 -U reviewdb -d reviewdb_test --no-owner --clean sql/reviewdb_test.pgdmp'
```
But, I got this error
```
% docker-compose run --rm console ./vendor/codeception/codeception/codecept -c console/codeception.yml run -g wip acceptance console/tests/acceptance/EMReportJob.feature
Creating review_console_run ... done
Codeception PHP Testing Framework v4.2.2 https://helpukrainewin.org
Powered by PHPUnit 8.5.29 #StandWithUkraine
[Groups] wip 

In Db.php line 680:

  Db: SQLSTATE[42501]: Insufficient privilege: 7 ERROR:  must be owner of schema public
```

Therefore, I implemented `the database is reset` to refresh the tables instead.